book/style_guide.md

```diff
@@ -0,0 +1,221 @@
+# Best practices
+
+This page lists our best practices we advice to follow. Keep in mind that they are not required to be
+used in external repositories (not ours), you can change them in the way you want,
+but please be consistent and follow your rules.
+
+## Formatting
+
+### Defaults
+
+**It's recommended to** assume that by default no spaces or tabs allowed, but the following rules define where they are allowed.
+
+### Basic
+
+- **It's recommended to** put one space before and after pipe `|` symbol, commands, subcommands, their options and arguments.
+- **It's recommended to** omit commas between list items.
+
+Correct:
+
+```nushell
+'Hello, Nushell! This is a gradient.' | ansi gradient --fgstart '0x40c9ff' --fgend '0xe81cff'
+```
+
+Incorrect:
+
+```nushell
+'Hello, Nushell! This is a gradient.' |  ansi gradient --fgstart '0x40c9ff' --fgend '0xe81cff' # two many spaces after "|": 2 instead of 1
+```
```

Comment:
to avoid having the scroll horizontally to see the comment, what about a simpler
> Correct:
>
> ```nushell
> ls | get name
> ```
>
> Incorrect:
>
> ```nushell
> # two many spaces after "|": 2 instead of 1
> ls |  get name
> ```
?

Reply:
Fixed, moved all comments on separate lines and converted them to list format:

```
# - <comment1>
# - <comment2>
# - ...
```

Reply:
thanks :yum: